src/shared/components/case-editor/services/work-allocation.service.ts

```diff
@@ -27,15 +27,21 @@ export class WorkAllocationService {
    * Call the API to get tasks matching the search criteria.
    * @param searchRequest The search parameters that specify which tasks to match.
    */
-  public searchTasks(searchRequest: TaskSearchParameters): Observable<object> {
-    const url = `${this.appConfig.getWorkAllocationApiUrl()}/task`;
+  public searchTasks(searchRequest: TaskSearchParameter): Observable<object> {
+    const url = `${this.appConfig.getWorkAllocationApiUrl()}/searchForCompletable`;
     return this.http
-      .post(url, { searchRequest })
+      .post(url, { searchRequest }, null, false)
       .pipe(
         map(response => response),
         catchError(error => {
           this.errorService.setError(error);
-          return throwError(error);
+          // explicitly eat away 401 error and 400 error
+          if (error && error.status && (error.status === 401 || error.status === 400)) {
```

Comment:
`error.status` (the second condition) is unnecessary considering the third and fourth conditions).